spec/models/post_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require 'faker'
+
+RSpec.describe Post, type: :model do
+  # VALID user
+  user = User.create(name: 'Jonh', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+  # VALID post
+  post = Post.create(author_id: user.id, title: 'Hello', text: 'This is my first post')
+
+  context 'is created with proper key and values, passes all validations and it' do
+    it 'should have a key "title" = Hello' do
+      expect(post.title).to eql('Hello')
+    end
+
+    it 'should have a key "text" = This is my first post' do
+      expect(post.text).to eql('This is my first post')
+    end
+
+    it 'should belong to a user with "name" = Jonh' do
+      expect(post.author.name).to eql('Jonh')
+    end
+
+    it 'should have a key "comments_counter" = 0' do
+      expect(post.comments_counter).to eql(0)
+    end
+
+    it 'should have a key "likes_counter" = 0' do
+      expect(post.likes_counter).to eql(0)
+    end
```

Comment:
- [ ] Kindly add a test case to check for the likes counter to be an integer greater than or equal to 0, as this is a requirement. Currently it only checks if it is equal to 0

Reply:
I've done it!

Thanks for pointing that out!